### DISCUSSION TITLE
Inspecting remote images

### DISCUSSION THREAD

**Discussion by tarun360** [containers/podman#15529] [Q&A] [answered]
I want to inspect a image stored in a registry without actually pulling the entire image. This would save time and bandwidth which comes with pulling an entire image, if you only want to get some meta-data of the image.

I am aware that [skopeo-inspect](https://man.archlinux.org/man/community/skopeo/skopeo-inspect.1.en) does provide this functionality. But I wanted to do it from Podman itself.

Thanks,
Tarun

**Comment by mheon** [comment, marked as answer]
@rhatdan @vrothberg WDYT?

**Reply by vrothberg** [reply]
I don't see it as a podman feature.  In fact, remote inspection was the first feature of skopeo :)  But I understand that the lines are blurry.

The outputs of `podman inspect` and `skopeo inspect` are very different as well.

**Reply by rhatdan** [reply]
Well skopeo was created because `docker inspect --remote` was rejected.  I don't love forcing people to install two tools to do very similar things.
